### ISSUE TITLE
In-chat sound is played for muted chats

### ISSUE THREAD

**Issue by lk108** [deltachat/deltachat-desktop#6224]
- Operating System (Linux/Mac/Windows/iOS/Android): Linux (.deb)
- Delta Chat Version: 2.48.0
- Expected behavior: No sound is played for muted chats
- Actual behavior: Incoming messages for the currently open chat trigger a sound even if the chat is muted. On DC Android, muting a chat not only suppresses notifications, but also in-chat sound.

**Comment by WofWca** [comment]
Related: https://github.com/deltachat/deltachat-desktop/issues/5766

This is intentional, for accessibility reasons. If a chat is open then we want to notify the user of incoming messages, same as we do with sighted users: we show the in-chat scroll-to-bottom badge, and scroll to the bottom when new messages arrive.
Example use case: you're at work and you don't want to be distracted by the memes your jobless friend sends you, so you mute the chat. But when at a break, you have some time for a chat so you open it and chat as usual, without missing their replies while writing your own texts.

Did you file this issue simply because this looks like a bug and is different from how it is on Android or do you think that it would actually be better not to play the sounds? I agree that it's intuitive that the chat wouldn't make sounds, but still.

I can see an "excuse" to mute also these in-chat sounds: if a chat is muted then the user indeed doesn't care if new messages arrive. When the user opens a chat, they are able to read the existing messages. If a new message arrives while they're in the chat, they will of course not be notified, but they can re-read the messages manually.

**Comment by lk108** [comment]
My apologies for not searching the closed issues.

> Did you file this issue simply because this looks like a bug and is different from how it is on Android

I'm afraid that this was the reason, yes.

As already elaborated by you, there are valid arguments for both suppressing and playing the sound in muted chats, but I'm fine with the decision taken in #5766. The only thing that bothers me somewhat is that it's different on Android.

**Comment by nicodh** [comment]
Seems there is nothing to do for now
